core/peer/peer.go

```diff
@@ -297,6 +302,14 @@ func Initialize(
 
 		InitChain(cid)
 	}
+
+	//initialize channel initializer
+	channelInitializer = xpeer.GetChannelInitializer(sccp, pm, deployedCCInfoProvider, legacyLifecycleValidation, newLifecycleValidation)
+}
+
+// InitializeChannel initializes the given channel from persistence
+func InitializeChannel(cid string) error {
```

Comment:
Not sure what the difference is between InitChain and InitializeChannel?

Reply:
InitializeChannel initializes channel from persistence, very similar to peer.Initialize().

But this function is removed after refactoring in recent patch.